tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/GetOrderRequestSteps.cs

```diff
@@ -85,6 +85,12 @@ public void ThenTheGetOrderResponseContainsYearlyValue(decimal amount)
             getOrderResponse.AssertOrderItemCost(amount);
         }
 
+        [Then(@"the get order response contains recipient with (.*) a yearly value of (.*)")]
+        public void ThenTheGetOrderResponseContainsRecipientWithAYearlyValueOf(string odsCode, decimal amount)
+        {
```

Comment:
As per the comments above, I think it might be better if we take a catalogue item ID parameter here.

Reply:
Added `catalogueItemID` parameter in step method.